processor/src/main/java/com/linecorp/decaton/processor/runtime/ProcessorSubscription.java

```diff
@@ -167,7 +159,11 @@ public void run() {
                 @Override
                 public void onPartitionsRevoked(Collection<TopicPartition> partitions) {
                     waitForRemainingTasksCompletion(rebalanceTimeoutMillis.value());
-                    commitCompletedOffsets(consumer);
+                    try {
+                        commitCompletedOffsets(consumer);
+                    } catch (CommitFailedException | TimeoutException e) {
+                        logger.warn("Offset commit failed at group rebalance", e);
```

Comment:
[nits] better to mention that process will be duplicated I think

Reply:
I think no, because of following reasons:

* Occurrence of this log doesn't mean there will be duplicate exactly, because everything's fine if it succeeds to commit the offset equals to or larger than the failed offset
* Explaining such complex situation like when will it be duplicate and when will not precisely in log isn't easy, as in I think too long log line is also not good.
* duplicate processing is widely recognized property of Kafka consumers, so I think users should be aware of such possibility at any cases, making it pointless to mention about it particularly here

Reply:
> Occurrence of this log doesn't mean there will be duplicate exactly, because everything's fine if it succeeds to commit the offset equals to or larger than the failed offset on next round

nevermind about this. This is at rebalance listener so the duplicate will happen absolutely.

Reply:
agree with point 3 "duplicate processing is widely recognized property of Kafka consumers"